packages/back-end/features/GetPerson.feature

```diff
@@ -0,0 +1,53 @@
+Feature: get personn information from the database
+
+
+  As an indentified person I want to get a person
+
+
+  Background:
+    Given a running app
+    And a clean database
+    And a person with data
+      """
+      {
+      id:1,
+      name: "John Doe"
+      }
+      """
+    And an authenticated user
+
+  Scenario: getting an existing personn information by id
+    When I visit 'getPerson/id/1'
+    Then I should see a 200 status code
+    And I should see a "person" field containing
+      """
+      {
+      id:1,
+      name: "John Doe"
+      }
+      """
+
+  Scenario: getting an invalid personn information by id
+    When I visit 'getPerson/id/2'
+    Then I should see a 404 status code
+    And I should see a "error" field containing "Person not found"
```

Comment:
Wouldn't a message field make more sense? The 404 implies that it's an error already

Reply:
Or maybe an empty list? easier to handle on a page.

Reply:
when specifying an id, we want a specific person so a list doesn't really make sense